Pod/Classes/RealexRemote/RealexRemote.swift

```diff
@@ -177,8 +170,8 @@ class RealexRemote: NSObject {
                 return false
             }
 
-            let month = Int(date.substring(with: (date.characters.index(date.startIndex, offsetBy: 0) ..< date.characters.index(date.startIndex, offsetBy: 2))))
-            let year = Int(date.substring(with: (date.characters.index(date.startIndex, offsetBy: 2) ..< date.characters.index(date.startIndex, offsetBy: 4))))
+            let month = Int(String(date[..<date.index(date.startIndex, offsetBy: 2)]))
+            let year = Int(String(date[date.index(date.startIndex, offsetBy: 3)...]))
```

Comment:
Is this offset correct? Also the old implementation would only return 2 characters whereas this could return more if the string is longer

Reply:
Got indexes wrong first time anyway